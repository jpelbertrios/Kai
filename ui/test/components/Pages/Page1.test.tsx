import {mount, shallow} from 'enzyme';
import React from 'react';
import '@testing-library/jest-dom/extend-expect'
import Page1 from "../../../src/components/Page1";

describe('When Page1 mounts', () => {
    it('should render without errors', () => {
        const component = shallow(<Page1/>);

        expect(component).toMatchSnapshot();
    });
});