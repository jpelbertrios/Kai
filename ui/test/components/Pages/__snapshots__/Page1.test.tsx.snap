// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Page1 mounts should render correctly in "debug" mode 1`] = `
<WithStyles(ForwardRef(Grid))
  className="makeStyles-root-1"
  container={true}
  justify="center"
>
  <WithStyles(ForwardRef(Paper))>
    <ExampleTable />
  </WithStyles(ForwardRef(Paper))>
</WithStyles(ForwardRef(Grid))>
`;

exports[`When Page1 mounts should render without errors 1`] = `
<WithStyles(ForwardRef(Grid))
  className="makeStyles-root-1"
  container={true}
  justify="center"
>
  <WithStyles(ForwardRef(Paper))>
    <ExampleTable />
  </WithStyles(ForwardRef(Paper))>
</WithStyles(ForwardRef(Grid))>
`;
